website/docs/Use-Cases/agent_chat.md

```diff
@@ -72,12 +72,13 @@ On the one hand, one can achieve fully autonomous conversations after an initial
 AutoGen, by integrating conversation-driven control utilizing both programming and natural language, inherently supports dynamic conversations. This dynamic nature allows the agent topology to adapt based on the actual conversation flow under varying input problem scenarios. Conversely, static conversations adhere to a predefined topology. Dynamic conversations are particularly beneficial in complex settings where interaction patterns cannot be predetermined.
 
 1. Registered auto-reply
-With the pluggable auto-reply function, one can choose to invoke conversations with other agents depending on the content of the current message and context. A working system demonstrating this type of dynamic conversation can be found in this code example, demonstrating a [dynamic group chat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat.ipynb). In the system, we register an auto-reply function in the group chat manager, which lets LLM decide who the next speaker will be in a group chat setting.
+With the pluggable auto-reply function, one can choose to invoke conversations with other agents depending on the content of the current message and context. A working system demonstrating this type of dynamic conversation can be found in this code example, demonstrating a [dynamic group chat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat.ipynb). In the system, we register an auto-reply function in the group chat manager, which lets LLM decide who the next speaker will be in a group chat setting. For example:
+- Hierarchical chat like in [OptiGuide](https://github.com/microsoft/optiguide).
+- Dynamic Group Chat which is a special form of hierarchical chat.
+- Graph based group chat which is a special form of dynamic group chat. In this approach, a directed graph is fed into [GraphGroupChat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_two_users.ipynb), an inherited class of `GroupChat`. `GraphGroupChat` takes the following steps in sequence to determine the next speaker: (1) It seeks a speaker suggestion from the previous speaker, thereby reducing API calls. (2) If the suggestion from the previous step is not valid, it utilizes the LLM API to nominate the next speaker amongst the eligible speakers. (3) If the suggestion from the previous step is not valid, it randomly selects from eligible speakers, defined as successor nodes (agents) from the prior speaker (agent). A valid suggestion is a transition path that is specified in the directed graph.
```

Comment:
The link looks wrong.

Reply:
😓 Sorry, I think you meant [GraphGroupChat] link was wrong, I have corrected it.